targets/targets.json

```diff
@@ -2826,6 +2826,29 @@
             "network-default-interface-type": "ETHERNET"
         }
     },
+    "MTB_STM_S2LP": {
+        "inherits": ["FAMILY_STM32"],
+        "core": "Cortex-M4F",
+        "config": {
+            "clock_source": {
+                "help": "Mask value : USE_PLL_HSE_EXTC | USE_PLL_HSE_XTAL (8MHz) | USE_PLL_HSI",
+                "value": "USE_PLL_HSE_XTAL",
+                "macro_name": "CLOCK_SOURCE"
+            },
+            "clock_source_usb": {
```

Comment:
Note that in ST boards, we don't use this config since #6776

Reply:
Thanks @jeromecoutant . Now removed and added "USBDEVICE"